packages/thirdweb/src/utils/address.ts

```diff
@@ -106,11 +106,11 @@ export function getAddress(address: string): Address {
  * import { shortenAddress } from 'thirdweb/utils';
  *
  * shortenAddress('0x5aAeb6053F3E94C9b9A09f33669435E7Ef1BeAed');
- * //=> '0x5aAe...Ef1BeAed'
+ * //=> '0x5a...eAed'
  * ```
  * @utils
  */
-export function shortenAddress(address: Address, length = 4) {
+export function shortenAddress(address: string, length = 4) {
```

Comment:
I meant it should leave 4 chars after the 0x on the first slice, like

0xabcd....1234